README.md

```diff
@@ -41,13 +45,15 @@ status = "generate"
 
 ### Generation in FFI mode
 
-When you're ready, let's generate the FFI part:
+When you're ready, let's generate the FFI part. In the command we'll execute, `../gir-files` is where we cloned the [gir-files repository](https://github.com/gtk-rs/gir-files). You need to put your Gir file in this repository as well to make your like easier. Then let's run the command:
```

Comment:
"your LIVE easier" ?
IMHO this better be rephrased to be lesser depend from [gir-files repository], something like "directory with your .gir files".
Also https://github.com/gtk-rs/gir-files already mentioned in introduction